Dockerfile

```diff
@@ -5,6 +5,10 @@ RUN wget -qO- https://deb.nodesource.com/gpgkey/nodesource.gpg.key | apt-key add
 RUN echo 'deb https://deb.nodesource.com/node_0.12 jessie main' > /etc/apt/sources.list.d/nodesource.list
 RUN apt-get update && apt-get install -y nodejs
 
+ENV GEM_HOME=/bundle
+RUN gem update --system
+RUN gem install bundler
```

Comment:
zdi automatically sets the `GEM_HOME` to `/bundle` and also runs `bundle check` when running an app with the `-d` option - I was getting errors about bundler not being found because of this. This project is different to others because it uses the Docker hub's (Debian) base Ruby image rather than our internal Ubuntu one.